### ISSUE TITLE
PLS take this repo down

### ISSUE THREAD

**Issue by DeusOmmuim** [Salty-Coder/Unblocked-Sites#3]
Me and Trickjoker312 were the orginal Linkers 
But As I got banned on my Main account Realhack1
I don't have a following anymore so I need you to take down this repo
No hard Feelings if you don't.

Thank You
from
Realhack1

**Comment by Deus-Scissuras** [comment]
Your too much of a competitor too me

**Comment by Salty-Coder** [comment]
Hello.
The sites in this repository are not created by you, Deus-Scissuras/Realhack1, or Trickjoker312.
I have found lists of websites from repositories like yours, and added additional sites from my own knowledge.
Furthermore, I have given Realhack1 and Trickjoker312 credit for the links referenced from your repositories.

You didn't have any objection to me referencing and crediting the links from your repository before, but now that you have less followers than you used to, you're telling me I have to take down the entire repository?
Is this really necessary?

I created this repository as a way to collaborate with other creators, such as yourself, to aim to make an ever-growing list of unblocked sites. I made sure to credit everyone who I have referenced links from. This repository was created for collaboration, not competition.
Does you really feel that being credited in this repository is having a negative impact on you?


And how did you get banned anyway?


-Salty-Coder
